devicetypes/smartthings/smartsense-button.src/smartsense-button.groovy

```diff
@@ -174,6 +177,19 @@ private Map translateZoneStatus(ZoneStatus zs) {
     } else { }
 }
 
+private Map translateMultiStatus(String description) { 
+    def descMap = zigbee.parseDescriptionAsMap(description)
+	log.debug "descMap.value is : ${descMap.value}"
+    
+    if (descMap.clusterInt == 0x0012 && descMap.value == "0002" ) {
+        return getButtonResult('double')
+    } else if (descMap.clusterInt == 0x0012 && descMap.value == "0001" ) {
+        return getButtonResult('pushed')
+    } else if (descMap.clusterInt == 0x0012 && descMap.value == "0000"){
+        return getButtonResult('held')
+    } else {}
```

Comment:
`clusterInt == 0x0012` is already true in all of these cases

Why are you re-parsing the description instead of just passing the value you need?